media/CfgEditor/index.html

```diff
@@ -0,0 +1,302 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script nonce="${nonce}" type="module" src="${toolkitUri}"></script>
+    <script nonce="${nonce}" type="module" src="${jsUri}"></script>
+    <link rel="stylesheet" href="${cssUri}">
+    <link rel="stylesheet" href="${codiconUri}">
+    <title>cfg editor</title>
+</head>
+<body>
+    <div class="maintitle">
+        <span class="codicon codicon-megaphone"></span>
+        <span>Click checkbox to enable each step.</span>
+    </div>
+    <div class="statusbar">
+        <div class="detail">
+            <div class="expanded" id="expandedDetail">
+                <span>Steps</span>
+            </div>
+        </div>
+        <div class="steps" id="statusbarSteps">
+            <div class="step" id="stepbarImport">
+                <vscode-checkbox id="checkboxImport"></vscode-checkbox>
+                <span id="stepImport" class="title">Import</span>
+            </div>
+            <div class="step" id="stepbarOptimize">
+                <vscode-checkbox id="checkboxOptimize"></vscode-checkbox>
+                <span id="stepOptimize" class="title">Optimize</span>
+            </div>
+            <div class="step" id="stepbarQuantize">
+                <vscode-checkbox id="checkboxQuantize"></vscode-checkbox>
+                <span id="stepQuantize" class="title">Quantize</span>
+            </div>
+            <div class="step" id="stepbarCodegen">
+                <vscode-checkbox id="checkboxCodegen"></vscode-checkbox>
+                <span id="stepCodegen" class="title">Codegen</span>
+            </div>
+            <div class="step" id="stepbarProfile">
+                <vscode-checkbox id="checkboxProfile"></vscode-checkbox>
+                <span id="stepProfile" class="title">Profile</span>
+            </div>
+        </div>
+    </div>
+    <div>
+        <vscode-divider></vscode-divider>
+    </div>
+    <div class="optionPanel">
+        <div class="options" id="optionSummary">
+            <div class="basic">
+                <div class="title">
+                    What Should We Display Here?
+                </div>
+                <div class="option">
+                    <vscode-text-field startIcon=true size=50 placeholder="">
+                        Some item
+                        <span slot="end" class="codicon codicon-search" style="cursor: pointer"></span>
+                    </vscode-text-field>
+                </div>
+            </div>
+        </div>
+        <div class="options" id="optionImport">
+            <div class="prerequisite">
+                <div class="title">
+                    Basic Options
+                </div>
+                <div class="option">
+                    <vscode-radio-group id="importInputModelType">
+                        <label slot="label">Input Model Type</label>
+                        <vscode-radio value="pb" checked="true">pb</vscode-radio>
```

Comment:
TODO / Main branch bugfix

```suggestion
                        <vscode-radio value="pb">pb</vscode-radio>
```